specification/specification.yml

```
@@ -0,0 +1,751 @@
+openapi: 3.1.0
```

[Редактор](https://editor.swagger.io/) выдает ошибку:

```
Structural error at openapi
should match pattern "^3\.0\.\d(-.+)?$"
pattern: ^3\.0\.\d(-.+)?$
Jump to line 1
```

```suggestion
openapi: 3.0.1
```